gluster_brick_provisioning.adoc

```diff
@@ -0,0 +1,41 @@
+// vim: tw=79
+= Gluster brick provisioning
+
+This document describes some of the conventions followed when tendrl provisions
+bricks for gluster volumes. Also it provides guidelines that user has to follow
+if he creates brick manually and tendrl has to pick that.
```

Comment:
No, We will be able to import cluster with any type of bricks.
This convention is for raw bricks( the bricks that are not consumed by any volume yet ). If user has created any such raw bricks, If he wants to see them in UI so that he can add them to a volume. Then he should update "gluster_bricks_dir"  attribute in gluster_integration conf file with path under which he has created raw bricks. Because tendrl considers mounts under only this directory as raw bricks